controllers/todo.js

```diff
@@ -23,6 +23,28 @@ const Todos = {
     });
   },
 
+  updateTodo(req, res) {
+    const { id } = req.params;
+    const { title, completed } = req.body;
+    Todo.findById(id).then((todo) => {
```

Comment:
Is the Todo title attribute unique? If no, then LGTM. Else, you'd need a block to handle when `todoFound` and to return a 409 conflict status in the case that that happens.

Reply:
Refactored !